config/services/node-exporter.service

```diff
@@ -0,0 +1,50 @@
+[Unit]
+Description=Prometheus Node Exporter
+Requires=network-online.target
+After=network-online.target
+
+[Service]
+StandardOutput=journal+console
+StandardError=journal+console
+Type=simple
+
+ExecStart=/usr/local/bin/node_exporter \
+    --web.listen-address 127.0.0.1:9000 \
+    --web.telemetry-path /metrics/node \
```

Comment:
To make grokking this unit easier, can you explicitly enable any collectors you want to collect? Just for readability's sake.